tests/recipes/cpc2/test_evaluate_cpc2.py

```diff
@@ -30,6 +31,8 @@ def test_rmse_score_ok(x, y, expected):
 def test_rmse_score_error(x, y, expected):
     """Test the function rmse_score for invalid inputs"""
     with pytest.raises(expected):
+        np.seterr(all="ignore")
+        warnings.simplefilter("ignore")  # <--- suppress mean of empty slice warning
```

Comment:
you can catch the warnings with the context manager. this will deactivate and reactivate the warnings
```
with warnings.catch_warnings():
   warnings.simplefilter("ignore")
```

Reply:
Thanks, this looks better.